src/pybind/mgr/dashboard/frontend/src/app/shared/api/mgr-module.service.ts

```diff
@@ -134,14 +138,17 @@ export class MgrModuleService {
     } else {
       $obs = this.enable(module);
     }
+
     $obs.subscribe(
-      () => undefined,
+      () => {
+        this.updateCompleted$.next();
+      },
       () => {
         // Suspend showing the notification toasties.
         this.notificationService.suspendToasties(true);
         // Block the whole UI to prevent user interactions until
         // the connection to the backend is reestablished
-        this.blockUI.start('global', $localize`Reconnecting, please wait ...`);
+        this.blockUI.start('global', reconnectingMessage);
```

Comment:
```suggestion
        this.blockUI.start('global', reconnectingMessage);
```
`global` can now be promoted to a constant :)